2.3/configuration/the-command-line-interface.md

```diff
@@ -0,0 +1,69 @@
+# The command line interface (CLI)
+
+[[toc]]
+
+## Run the CLI
+
+```bash
+ls-lint [options] [file|dir]*
+```
+
+### Options
+
+ls-lint provides multiple CLI options out of the box:
+
+| Option                                | Description                                                    |
+|---------------------------------------|----------------------------------------------------------------| 
+| [--config](#extending-configurations) | ls-lint config file path(s)                                    |
+| --workdir                             | change working directory before executing the given subcommand |
+| --error-output-format                 | use a specific error output format (text, json)                |
+| --debug                               | write debug informations to stdout                             |
+| --warn                                | write lint errors to stdout instead of stderr (exit 0)         |
+| --version                             | prints version information for ls-lint                         |
+
+:::tip Help
+`ls-lint --help` shows up all available options
+:::
+
+### Extending configurations
+
+The `--config` option provides a way to specify a custom path for your ls-lint configuration yml or yaml file:
+
+```bash
+ls-lint --config .ls-lint.(yml|yaml)
+```
+
+You can specify _`N`_ `--config` options than can be merged together.
```

Comment:
```suggestion
You can specify multiple `--config` options than can be merged together.
```

A lot of developers dislike Mathematics, and don't understand n, n²…

Reply:
mh